app/lib/actions/katello/host/register.rb

```diff
@@ -46,6 +44,14 @@ def humanized_name
           end
         end
 
+        def run
+          host = ::Host.find(input[:host_id])
+          unless input[:facts].blank?
+            ::Katello::Host::SubscriptionFacet.update_facts(host, input[:facts])
+            input[:facts] = 'TRIMMED'
```

Comment:
It is, the purpose is so that we do not cause the foreman tasks tasks table We do that in other places such as:

https://github.com/Katello/katello/blob/a1b1df1dc0f73aef92ce92df40924bb2c7521694/app/lib/actions/candlepin/consumer/update.rb#L15